Makefile

```diff
@@ -1,27 +1,25 @@
 .PHONY: test clean build
 
-VENDOR_FLAG = GO15VENDOREXPERIMENT=1
-GO_CMD = $(VENDOR_FLAG) godep go
```

Comment:
`GO15VENDOREXPERIMENT=1` isn't needed anymore. It got removed after go 1.6. Also since go's vendor is supported by default, we don't need to use `godep go` anymore.